tests/test_plot_dag.py

```diff
@@ -310,3 +309,18 @@ def test_params_are_removed_from_dag(include_params, expected_nodes):
         include_params=include_params,
     )
     assert set(dag.nodes()) == set(expected_nodes)
+
+
+def test_orphaned_policy_inputs_are_removed_from_dag():
```

Comment:
```suggestion
def test_orphaned_dates_are_removed_from_dag():
```
We don't want to remove all orphans, right?